index.html

```diff
@@ -348,50 +356,52 @@ <h2 class="text-white font-weight-bold py-2">Contact Us</h2>
                 </div>
             </div>
         </div>
-        <div class="container mt-5 pt-lg-5">
-            <div class="row">
-                <div class="col-md-3 col-sm-6 col-lg-3 col-12">
-                    <div class="">
-                        <img src="./images/icon.png" alt="" class="img-fluid mb-lg-4" width="100" height="100">
-
-                    </div>
-                    <p class="text-white">An Open Source organization focuses on Data Structure and Algorithms.</p>
-                </div>
-                <div class="col-lg-2 col-md-2 col-sm-6 col-12 offset-lg-1">
+        <div class="container col-lg-16 col-lg-offset-16" >
+            <div class="row" style="margin-left:20%;">
+                
+                <div class=" center col-lg-4.5 col-md-2 col-sm-6 col-12 offset-lg-1">
                     <h5 class="font-weight-bold text-white mb-lg-4 pb-2 margin-top-mobile">Useful Links</h5>
                     <ul>
                         <li>
                             <a class="text-white mb-lg-3" href="#about">About</a>
                         </li>
+                        <hr style="margin-right: 40%;">
                         <li>
                             <a class="text-white mb-lg-3" href="./team.html">Our Team</a>
                         </li>
+                        <hr style="margin-right: 40%;">
                         <li>
                             <a class="text-white mb-lg-3" href="#footer">Contact</a>
                         </li>
+                        <hr style="margin-right: 40%;">
                         <li>
                             <a class="text-white mb-lg-3" href="./sitemap.html">SiteMap</a>
                         </li>
+                        <hr style="margin-right: 40%;">
                     </ul>
                 </div>
 
-                <div class="col-lg-3 col-md-3 col-sm-6 col-12">
+                <div class="center col-lg-4.5 col-md-3 col-sm-6 col-12">
```

Comment:
There are no classes `center` and `col-lg-4.5` either in bootstrap or in custom styles.